staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -210,7 +210,8 @@ func (le *LeaderElector) Run(ctx context.Context) {
 }
 
 // RunOrDie starts a client with the provided config or panics if the config
-// fails to validate.
+// fails to validate. RunOrDie blocks until leader election loop is 
+// is stopped by ctx.
```

Comment:
add similar documentation to `Run`?

Reply:
Updated @liggitt . Let me know what you think.